src/main/java/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration.java

```diff
@@ -205,7 +205,7 @@ private static FormValidation checkBaseUrl(String baseUrl) {
      */
     private static FormValidation checkServerName(String serverName) {
         return isBlank(serverName)
-                ? FormValidation.error("Enter a name to help users identify this instance.")
+                ? FormValidation.error("Required")
```

Comment:
Same reasoning as before. Because the user sees the label, help text, and validation all at once I think it is better to simplify here and to leave this as is.